GpxLocationManager/LocationManager.swift

```diff
@@ -200,7 +200,25 @@ open class LocationManager {
     }
   }
 
-  open let locationManagerType: LocationManagerType
+  public let locationManagerType: LocationManagerType
```

Comment:
Question: Are you using the `LocationManager` wrapper in your code, as `RaceRunner`, the app that prompted me to create `GpxLocationManager`, does, or are you using `GpxLocationManager` directly?